### ISSUE TITLE
keep carshes every game even when i open it after the loading screen, it was working last patch but idk what happend now

### ISSUE THREAD

**Issue by 6oody** [B3akers/LeagueSkinChanger#103]

**Comment by 6oody** [comment]
if anyone can help tell me

**Comment by thornar** [comment]
are you using updated version?
did not encounter any crash with https://anonfiles.com/T9C9i38eu4/LeagueSkinChanger.v11.5_rar

**Comment by 6oody** [comment]
> are you using updated version?
> did not encounter any crash with https://anonfiles.com/T9C9i38eu4/LeagueSkinChanger.v11.5_rar

I tried this and tried another one and still crashes every time i just join the game after the loading screen

**Comment by R3nzTheCodeGOD** [comment]
### **hello @6oody** 
I have redeveloped the project. You can recompile or download the project from the link below and use it, it should not crash anymore

Project Link: https://github.com/R3nzTheCodeGOD/R3nzSkin

**Comment by 6oody** [comment]
> ### **hello @6oody**
> I have redeveloped the project. You can recompile or download the project from the link below and use it, it should not crash anymore
> 
> Project Link: https://github.com/R3nzTheCodeGOD/R3nzSkin

Thanks man, i appreciate it so much.
i will use after my internet back ' in like 10 days unluckly me '
btw do u will update it in the future ?

**Comment by R3nzTheCodeGOD** [comment]
> > ### **hello @6oody**
> > I have redeveloped the project. You can recompile or download the project from the link below and use it, it should not crash anymore
> > Project Link: https://github.com/R3nzTheCodeGOD/R3nzSkin
> 
> Thanks man, i appreciate it so much.
> i will use after my internet back ' in like 10 days unluckly me '
> btw do u will update it in the future ?

Yes, I will continue to improve, some skins have a bug that causes 1 or 2 crashes in the middle of the game, I will try to solve it

**Comment by 6oody** [comment]
> > > ### **hello @6oody**
> > > I have redeveloped the project. You can recompile or download the project from the link below and use it, it should not crash anymore
> > > Project Link: https://github.com/R3nzTheCodeGOD/R3nzSkin
> > 
> > 
> > Thanks man, i appreciate it so much.
> > i will use after my internet back ' in like 10 days unluckly me '
> > btw do u will update it in the future ?
> 
> Yes, I will continue to improve, some skins have a bug that causes 1 or 2 crashes in the middle of the game, I will try to solve it

Thanks again hommie ♥ 
I will share it with my friends 
and i will try to help u to find bugs if i found one
